src/models/prototype_model/encoders.py

```diff
@@ -176,25 +176,162 @@ def preprocess(self, x: torch.Tensor) -> torch.Tensor:
-
-                if isinstance(module.stride, tuple):
-                    features['strides'].append(module.stride[0])
-                else:
-                    features['strides'].append(module.stride)
-
-                if isinstance(module.padding, tuple):
-                    features['paddings'].append(module.padding[0])
-                else:
-                    features['paddings'].append(module.padding)
+        features['kernel_sizes'] = [
+            7,
+            3,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+            1,
+            3,
+            1,
+        ]
+        features['strides'] = [
+            2,
+            2,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            2,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            2,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            2,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+        ]
+        features['paddings'] = [
+            3,
+            1,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+            0,
+            1,
+            0,
+        ]
```

Comment:
@mak-en 

150 lines of code to create 3 variables?

Reply:
Autoformatting by Black.

Reply:
@ViacheslavDanilov